website/static/css/site.css

```diff
@@ -444,10 +444,16 @@ img.hg-addon-icon {
     display: block;
 }
 
+/* TODO: Remove this - it's far too broad */
```

Comment:
I don't want to remove it as part of this PR, because it will impact every page on the site. It's too big of a chunk :)